src/hooks/useEvidences.js

```diff
@@ -42,33 +42,49 @@ export default function useEvidences(rawEvidences) {
       return { ...baseEvidence, error: ERROR_TYPES.ERROR_FETCHING_IPFS }
     }
 
-    evidencesCache.current.set(id, data)
-    return {
+    const evidenceProcessed = {
       ...baseEvidence,
       metadata: data,
     }
+    evidencesCache.current.set(id, evidenceProcessed)
+
+    return evidenceProcessed
   }, [])
 
   useEffect(() => {
     let cancelled = false
 
     const updateEvidences = async () => {
-      const evidences = await Promise.all(rawEvidences.map(fetchEvidence))
-      if (!cancelled) {
-        setEvidences(
-          evidences.sort(
-            (evidenceA, evidenceB) => evidenceA.createdAt - evidenceB.createdAt
-          )
-        )
-      }
+      await Promise.all(
+        rawEvidences.map(async rawEvidence => {
```

Comment:
I guess we could remove the `Promise.all()` and use `forEach()` since we are not doing anything after they complete − or should we use it to export a `loading` boolean?

Reply:
maybe we can remove it since we are fetching one by one right?

Reply:
Yes sounds good!

Reply:
> maybe the sync indicator is also needed to let the user know that we are still fetching right?

Yes, this would be nice!

Reply:
<img width="1477" alt="Screen Shot 2020-03-03 at 11 12 32 PM" src="https://user-images.githubusercontent.com/11763623/75837757-95f36f00-5da4-11ea-86fb-2dabffdcde12.png">
@dizzypaty wondering which ` theme.floating` should we use here for the sync indicator

Reply:
Hey @rperez89 it seems to me that's the right color to use for the sync indicator. It's not a dark theme color, it's the default one because it needs to catch users attention as it's a feedback indicator, as it's also not persistent.